ghost/i18n/locales/ru/portal.json

```diff
@@ -104,8 +104,8 @@
     "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Убедитесь, что письма случайно не попадают в папки «Спам» или «Промоакции» вашего почтового ящика. Если это так, нажмите «Не спам!» и/или «Отметить как не спам» и/или «Переместить во входящие».",
     "Manage": "Управление",
     "Maybe later": "Может быть позже",
-    "Memberships from this email domain are currently restricted.": "",
-    "Memberships unavailable, contact the owner for access.": "Членство недоступно, свяжитесь с владельцем сайта для получения доступа.",
+    "Memberships from this email domain are currently restricted.": "В настоящее время количество подписчиков с этого домена электронной почты ограничено.",
+    "Memberships unavailable, contact the owner for access.": "Подписка недоступна, свяжитесь с владельцем сайта для получения доступа.",
```

Comment:
Ai says there's an error between singular and plural. Please review. (Feel free to tell me AI is wrong!)